fast_llm/functional/config.py

```diff
@@ -39,6 +39,7 @@ class ActivationType(enum.StrEnum):
     An enum for the available activation types for the MLP layer.
     """
 
+    gelu_gaussian = "gelu_gaussian"
```

Comment:
There is no point in having a separate gelu because we're not going for exact numerical match, and we already have bigger numerical differences anyway. HF gelu and gelu_pytorch_tanh should both map to our standard gelu